fintech-examples/fintech-api/src/main/resources/static/fintech_api.yml

```diff
@@ -33,17 +33,17 @@ info:
     Reloading the FinTechUI, we will also loose the XSRF parameter that is used to validate the SessionCookie. This is why we
     set RedirectCookie (that this time has a very short life span). The url reloading the FinTechUI must carry a redirectState
     parameter that will be used to invoke the /afterReload endpoint of the FinTechApi. Thus leading to a new SessionCookie and
-    corresponding XSRF parameter.     
+    corresponding XSRF parameter.
 
 
 externalDocs:
   url: "https://adorsys.github.io/open-banking-gateway"
-  termsOfService: urn:tos
-  contact: {}
-  license:
+  x-termsOfService: urn:tos
```

Comment:
why we need to put `x-` here? This tags are standard

Reply:
https://swagger.io/specification/#externalDocumentationObject
https://swagger.io/specification/#specificationExtensions
Here it's described, that we have to use x- properties.